sdk/search/Azure.Search.Documents/tests/DocumentOperations/VectorSearchTests.cs

```diff
@@ -150,6 +152,33 @@ await foreach (SearchResult<Hotel> result in response.GetResultsAsync())
                 "9", "3", "2", "5", "10", "1", "4");
         }
 
+        [Test]
+        [PlaybackOnly("The availability of Semantic Search is limited to specific regions, as indicated in the list provided here: https://azure.microsoft.com/explore/global-infrastructure/products-by-region/?products=search. Due to this limitation, the deployment of resources for weekly test pipeline for setting the \"semanticSearch\": \"free\" fails in the UsGov and China cloud regions.")]
+        public async Task SemanticMaxWaitOutOfRangeThrows()
+        {
+            await using SearchResources resources = await SearchResources.CreateWithHotelsIndexAsync(this);
+
+            RequestFailedException ex = await CatchAsync<RequestFailedException>(
+                async () => await resources.GetSearchClient().SearchAsync<Hotel>(
+                    "Is there any luxury hotel in New York?",
+                    new SearchOptions
+                    {
+                        SemanticSearch = new()
+                        {
+                            SemanticConfigurationName = "my-semantic-config",
+                            QueryCaption = new(QueryCaptionType.Extractive),
+                            QueryAnswer = new(QueryAnswerType.Extractive),
+                            MaxWait = TimeSpan.FromMilliseconds(700),
+                        },
+                        QueryType = SearchQueryType.Semantic,
+                    }));
+
+            Assert.AreEqual("InvalidRequestParameter", ex.ErrorCode);
+            StringAssert.StartsWith(
+                "The 'semanticMaxWaitInMilliseconds' parameter needs to be larger than 750.",
+                ex.Message);
+        }
```

Comment:
Why are we asserting service-specific behavior that can and will likely change? This seems to go against our [guidelines](https://azure.github.io/azure-sdk/general_implementation.html#general-params-server-validation).